### DISCUSSION TITLE
post/windows-sandboxie/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#20] [Announcements]
# post/windows-sandboxie/

日常使用 Windows 的过程中难免需要安装一些国产流氓软件(说的就是你腾讯)或者不常用的破解/来源不可信的软件. 或者要安装一些不常使用的软件, 但总会卸载不干净, 残留部分文件或注册表项在电脑中, 让洁癖和强迫症很不爽. 与现代的手机操作系统不同, PC 操作系统相对更加开放自由, 但同时也没有沙盒隔离机制, 导致软件权限不可控. 例如恶意软件能轻松地读取浏览器 Cookie 并上

https://blog.lyc8503.site/post/windows-sandboxie/

**Comment by JasonHe** [comment]
大佬，你GH上的那个SandboxiePlus破解版的项目还有可能继续更新吗？迫切需要，谢谢谢谢！
https://github.com/lyc8503/Sandboxie-crack

**Reply by lyc8503** [reply]
新版有什么重要的 feature 吗，我目前使用这个版本似乎没有遇到什么问题。Sandboxie 本身(为了卖许可证)更新的非常勤快，大多是小修小补，不更新一般问题不大。最近比较忙，暂时不会更新，之后有时间可能会跟进。
